index.html

```diff
@@ -21,7 +21,7 @@
   <li><a href="#portfolio">Portfolio</a></li>
   <li><a href="#about">About</a></li>
   <li><a href="#contact">Contact</a></li>
-  <li><img src="PImages/Icon - Mail.png" alt=""></li>
+  <li><img src="PImages/iconMail.png" alt=""></li>
```

Comment:
- [ ] Kindly add an alt attribute to this image. This image is not only for decoration but should be a link to write an email. It is therefore important that the alt attribute is appropriately named 👍🏾